source/comply/audit-log.rst

```diff
@@ -1,369 +1,290 @@
+|          |          | **Note**: Should only be used in testing environments, and shouldn’t be used in production environments.                        |
++----------+----------+---------------------------------------------------------------------------------------------------------------------------------+
+
+TCP target configuration options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++----------+----------+---------------------------------------------------------------------------------------------------------------------------------+
+| **Key**  | **Type** | **Description**                                                                                                                 |
++----------+----------+---------------------------------------------------------------------------------------------------------------------------------+
+| host     | string   | IP or domain name of the server receiving the log records.                                                                      |
++----------+----------+---------------------------------------------------------------------------------------------------------------------------------+
+| port     | number   | Port number for the server receiving the log records.                                                                           |
++----------+----------+---------------------------------------------------------------------------------------------------------------------------------+
+| tls      | bool     | Create a TLS connection to the server receiving the log records. Default is ``false``.                                          |
++----------+----------+---------------------------------------------------------------------------------------------------------------------------------+
+| cert     | string   | Path to a cert file (.pem) to be used when establishing a TLS connection to the server.                                         |
++----------+----------+---------------------------------------------------------------------------------------------------------------------------------+
+| insecure | bool     | Mattermost Boards accepts any certificate presented by the server, and any host name in that certificate. Default is ``false``. |
+|          |          | **Note**: Should only be used in testing environments, and shouldn’t be used in production environments.                        |
++----------+----------+---------------------------------------------------------------------------------------------------------------------------------+
+| tag      | string   | Syslog tag field.                                                                                                               |
++----------+----------+---------------------------------------------------------------------------------------------------------------------------------+
+
+Format configuration options
+----------------------------
+
+Plain log format configuration options
```

Comment:
We don't actually specify what the available format types are: `plain`, `json`, `gelf`    A few do appear in the samples.

Reply:
Correct. This isn't yet comprehensive logging documentation. This PR focuses on audit logging only. Docs has a card on the backlog to revisit logging documentation in general in a future release.